app/views/documents/show/_scheduling_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<% publish_datetime = @edition.revision.scheduled_publishing_datetime %>
+<% default_date = Time.zone.now.tomorrow %>
+
+<%= form_tag(save_scheduled_publishing_datetime_path(@edition.document)) do %>
+  <%= render "govuk_publishing_components/components/details", {
+    title: if publish_datetime
+             simple_format("Publish date: \n" + publish_datetime.strftime("%-d %B %Y - %l:%M%P"))
+           else
+             "Schedule publishing"
+           end
+  } do %>
+    <% if flash[:issues] %>
```

Comment:
I think we should be using the `error_summary` component here, as it looks like we're trying to summarise some errors - the checker framework is designed to be compatible with its interface, so it should be easier to slot in here, and gives better results with assistive tech if we do have multiple errors :-).

Reply:
To be honest, the error summary looks pretty horrendous in that small space.  I think it looks better visually to have just the errors listed out on their own.  I'll run it past Sonia when she's back in any case.

Reply:
That makes sense - @alex-ju, do you have any thoughts about how we could improve the screen reader experience here?

Reply:
My personal view is that we should use the error-summary placed at the top of the page - as we use it on other pages. It will point users to the input that needs to be reviewed so they won't have a problem in finding it in the current page. We may need to open the details element for them thought.. Let's see what are our designer's thoughts on this too.